### PR TITLE
Upgrade to Ubuntu 20.04 and ROS Noetic

### LINKED ISSUE #3307: ROS libraries
**Describe the Bug**
When launching the `e_puck_line.launch` ros example, I get the following error:

```
/snap/webots/x1/usr/share/webots/projects/default/controllers/ros/ros: error while loading shared libraries: libboost_system.so.1.65.1: cannot open shared object file: No such file or directory
WARNING: 'ros' controller exited with status: 127.
```

I had to manually add the ros libraries to the `LD_LIBRARY_PATH`:

export LD_LIBRARY_PATH=${WEBOTS_HOME}/lib/controller:`${WEBOTS_HOME}/projects/default/controllers/ros/lib/ros`

As I have not seen in the tutorial that it was needed to do so, I wonder if this is normal.

**System**
 - Ubuntu 20.04, Snap version of Webots

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Should fix https://github.com/cyberbotics/webots/issues/3307

My assumption is that the snap is compiled with the old ROS distribution which breaks usage with Noetic

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 name: webots
 title: Webots
-base: core18
+base: core20
```
Comment (lukicdarkoo) [inline snapcraft.yaml:14]: This wasn't necessary, but I believe it is good to stay updated

**Comment by lukicdarkoo** [comment]
I tested and it works locally fine with ROS.

@cyberbotics/maintainers It would be great for somebody to review it before tonight :)

**Review by omichel — APPROVED** [review]
Thank you.

File: snapcraft.yaml (hunk 1 above)
Comment (stefaniapedrazzi) [reply snapcraft.yaml:14]: Based on the documentation (https://snapcraft.io/docs/base-snaps) `core18` is "the currently recommended base for the majority of snaps".

**Pull request merged**